recipes/libfec/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "libfec" %}
+{% set version = "2.1.1+20161003" %}
```

Comment:
Please use a fully released version

Reply:
I don't think there is a tag for a release version but a git, right @ryanvolz?, and may be some patches for debian as we're basing it on that source. But maybe better to use the base version 2.1.1 in conda then?

Reply:
The whole thing is a mess. The 2.1.1 version appears in the README dated to 2006. There's a github tag for a "1.0" in Feb. 2016 whose purpose seems to be to hold a JavaScript file. You can try with that "1.0" tag since that technically applies to the whole repository, but it doesn't look like it has CMake support. You might be able to extract a patch for that though.

Debian is using the same git revision and versioning as in this PR, with some additional patches. Maybe we could consider the Debian version as the de-facto upstream at this point, and pull the tarball from there? It's rather annoying that even the weight of a big distribution wasn't enough to provoke an official release.

I'll also note here that I made a package for `libcorrect`, which is in theory a replacement for `libfec`, but it turned out to not support all of the features the `gr-satellites` requires without some significant hacking. The `gr-satellites` maintainers are probably disinclined to make that switch, since it seems the intention is to drop `libfec` when similar functionality gets built into `gnuradio` for the next major version. Maybe we're just between a rock and a hard place right now and need to wait it out.

Reply:
Thanks for commenting. Somehow it feels strange to use the 1.0 tag and thus version as the readme states 2.1.1... Is there a nomenclature in conda for post-2.1.1?

 @daniestevez, do you have some update about status of libfec and future versions?

Reply:
When https://github.com/gnuradio/gnuradio/pull/3657 is merged into GNU Radio, gr-satellites will no longer need libfec. However I think that even if this is merged into GNU Radio's master branch, it won't be realeased in the 3.8 series and only make it to the 3.9 series (which by the way breaks backwards compatibility again with OOT modules).

As a temporary workaround, maybe it is a good idea to copy over into gr-satellites the few Reed-Solomon functions from libfec that are needed. I haven't had much time to work into gr-satellites lately, but I've added https://github.com/daniestevez/gr-satellites/issues/148 as a self-reminder. I'll ping you here when this is done.

Reply:
Great, thanks!

Reply:
I have now made the changes to remove libfec in the [remove-libfec](https://github.com/daniestevez/gr-satellites/tree/remove-libfec) branch of gr-satellites. Is it easy to test from this branch if gr-satellites now can build on Windows or is it better that I merge this to master?

Reply:
@petrushy can temporarily point this PR's build to the new gr-satellites commit and enable the Windows build to see if it works. I'm not recalling at the moment exactly what it was that was preventing `libfec` from compiling on Windows in the first place, but if you only copied the necessary parts it's possible you're avoiding it.

Reply:
I think (but haven't tested) that the problem with building libfec on Windows might have to do with the detection of CPU capabilities, inline asm and so on that it uses for Viterbi. The Reed-Solomon code (which is what I copied) doesn't do anything fancy and it should be multiplatform.

Reply:
Hi! I will check if it builds on win!